webserver/views/datasets.py

```diff
@@ -167,10 +171,23 @@ def evaluate(dataset_id):
         try:
             if form.filter_type.data == forms.DATASET_EVAL_NO_FILTER:
                 form.filter_type.data = None
+            # take default value
+            # if invalid non-numerical c and gamma
+            try:
+                c_value = [int(value) for value in form.c_value.data.split(',')]
+            except:
```

Comment:
Instead of catching all exceptions, mentioning which error you're looking for here (`ValueError`) would be a good idea.